experimental/packages/opentelemetry-sdk-metrics-base/src/MeterProvider.ts

```diff
@@ -18,7 +18,7 @@ import * as api from '@opentelemetry/api';
 import * as metrics from '@opentelemetry/api-metrics';
 import { Resource } from '@opentelemetry/resources';
 import { Meter } from './Meter';
-import { MetricReader } from './export/MetricReader';
+import { MetricReader, ReaderForceFlushOptions, ReaderShutdownOptions } from './export/MetricReader';
```

Comment:
Hmm, should we rename those `ForceFlushOptions` and `ShutdownOptions`? :thinking:

Reply:
SGTM. Updated :)